dashboard/job-pulse-kanban.html

```
@@ -0,0 +1,418 @@
+         border-radius:6px;padding:9px 18px;font-size:.8rem;z-index:200;
+         pointer-events:none;max-width:min(600px,90vw);word-break:break-word;
+         animation:fadeout .4s ease 3.6s forwards}
+  @keyframes fadeout{to{opacity:0}}
+</style>
+</head>
+<body>
+<header>
+  <h1>⚡ Job Pulse</h1>
+  <span id="status">—</span>
+  <div class="toolbar">
+    <button class="primary" onclick="fetchJobs()">⬇ Fetch Jobs</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)